.github/workflows/import-check.yaml

```
@@ -0,0 +1,52 @@
+name: import-check
+
+on:
+  push:
+    branches: [develop, releases/**]
+  pull_request:
+    branches: [develop, releases/**]
```

```suggestion
on:
  workflow_call:
```

This should be triggered by a workflow_call instead similar to the other CI files and then called from `ci.yml` after the filtering job.